src/solids4FoamModels/Make/options

```diff
@@ -29,19 +29,23 @@ else
 endif
 
 ifdef PETSC_DIR
+    include ${PETSC_DIR}/lib/petsc/conf/petscvariables
     ifeq ($(WM_PROJECT), foam)
         include $(RULES)/mplib$(WM_MPLIB)
     else
         ifneq (,$(findstring v,$(WM_PROJECT_VERSION)))
             include $(GENERAL_RULES)/mpi-rules
         else
-            include $(GENERAL_RULES)/mplibType
+            -include $(GENERAL_RULES)/mplibType
```

Comment:
Thank you for these proposed changes.

Is the change at this line (line 39) a mistake? i.e., I guess `-` should not be added.

Reply:
It seems that this extra `-` is actually a typo/bug from a previous version of solids4foam; this is not used by OpenFOAM-v2012 so it did not affect anything for you. I will fix this in this branch and then merge the branch. Thanks!

Reply:
sorry for the late reply, just come back from AL. All good to go !

Reply:
Thanks again!